pkg/eks/components/Networking.vue

```diff
@@ -0,0 +1,240 @@
+<script lang="ts">
+import { _EDIT } from '@shell/config/query-params';
+import { defineComponent } from 'vue';
+import { Store, mapGetters } from 'vuex';
+import LabeledSelect from '@shell/components/form/LabeledSelect.vue';
+import Checkbox from '@components/Form/Checkbox/Checkbox.vue';
+import ArrayList from '@shell/components/form/ArrayList.vue';
+import { RadioGroup } from '@components/Form/Radio';
+
+export default defineComponent({
+  name: 'EKSNetworking',
+
+  components: {
+    LabeledSelect,
+    ArrayList,
+    Checkbox,
+    RadioGroup
+  },
+
+  props: {
+    mode: {
+      type:    String,
+      default: _EDIT
+    },
+    region: {
+      type:    String,
+      default: ''
+    },
+    amazonCredentialSecret: {
+      type:    String,
+      default: ''
+    },
+
+    subnets: {
+      type:    Array,
+      default: () => []
+    },
+    publicAccess: {
+      type:    Boolean,
+      default: false
+    },
+    privateAccess: {
+      type:    Boolean,
+      default: false
+    },
+    publicAccessSources: {
+      type:    Array,
+      default: () => []
+    },
+    rules: {
+      type:    Object,
+      default: () => {}
+    }
+  },
+
+  watch: {
+    amazonCredentialSecret: {
+      handler(neu) {
+        if (neu) {
+          this.fetchVpcs();
+        }
+      },
+      immediate: true
+    },
+    region: {
+      handler(neu) {
+        if (neu) {
+          this.fetchVpcs();
+        }
+      },
+      immediate: true
+    },
+
+    'chooseSubnet'(neu: boolean) {
+      if (!neu) {
+        this.$emit('update:subnets', []);
+      }
+    }
+
+  },
+
+  data() {
+    return {
+      loadingVpcs:  false,
+      vpcInfo:      {} as any,
```

Comment:
There's quite a lot of these around (search for `as any,`). Some could be defined as 
```
      vpcInfo:     {} as {
        // [key: string]: any,
        Vpcs: [],
      },
```
The commented out part could be brought it, if it needs to be more generic, but ideally not